### DISCUSSION TITLE
GitHub copilot two subscriptions

### DISCUSSION THREAD

**Discussion by 1alexvash** [community/community#147948] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hi there, I already have a GitHub Copilot subscription, and my new company is also paying for it. Should I do anything about this, or just leave it as is?

**Comment by Akash1134** [comment]
Hi @1alexvash ,

It depends on the usage of your Copilot subscription and your company policy. You are personally paying for GitHub Copilot, and it’s likely tied to your personal GitHub account. As your company’s subscription is providing Copilot access, it is possibly tied to your work account or covering the cost of Copilot for your personal account.

Now to what you should do:  

- If you’re comfortable using the company-paid subscription: Cancel your personal subscription to avoid duplicate charges. Note that they  might have specific guidelines on how to handle individual subscriptions versus company-provided ones.
- If you prefer to keep a personal subscription: Ensure your company isn’t paying for your usage, as this might violate their intent or reimbursement policies. 

Hope that helps!
